### PR TITLE
create, delete & validate users

### PR DESCRIPTION
**Pull request by fabian202**
startcasae

### CODE REVIEW

File: src/components/molecules/AdminTable/index.js [hunk 1]
```diff
@@ -37,23 +40,38 @@ const AdminTable = ({
...
       setPage(1)
     }
   }, [filter])
 
+  useEffect(() => {
+    if (deleteId) {
+      setShowConfirm(true)
+    } else {
+      setShowConfirm(false)
+    }
```
Comment (normandanielmq) [inline src/components/molecules/AdminTable/index.js:72]: `setShowConfirm(!!deleteId)`

File: src/components/molecules/AdminTable/index.js [hunk 2]
```diff
@@ -126,8 +150,8 @@ const AdminTable = ({
           <Table size="sm">
             <thead>
               <tr>
-                {columns.map(({ name, sortable }) => (
-                  <th>
+                {columns.map(({ name, sortable, key }) => (
+                  <th className={classnames({ 'd-none': key })}>
```
Comment (normandanielmq) [inline src/components/molecules/AdminTable/index.js:154]: Missing key property. Please do Monitoring browser's console, it should be a warning there about this

File: src/components/molecules/AdminTable/index.js [hunk 3]
```diff
@@ -138,8 +162,13 @@ const AdminTable = ({
             <tbody>
               {paginated?.map((d) => (
                 <tr>
-                  {columns.map(({ name }) => (
-                    <td title={d?.[name]}>{d?.[name]}</td>
+                  {columns.map(({ name, key }) => (
+                    <td
```
Comment (normandanielmq) [inline src/components/molecules/AdminTable/index.js:166]: Missing key property

File: src/components/organisms/UserForm/index.js [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+              ) : (
+                <>
+                  <FaPen className="mr-1" /> Create
+                </>
+              )}
+            </Button>
+            <Button variant="secondary" type="button" className="m-2 px-3 py-2">
+              <FaTimes className="mr-1" />
+              Cancel
+            </Button>
```
Comment (normandanielmq) [inline src/components/organisms/UserForm/index.js:171]: Use the CancelButton component

**Review by cesar82 — APPROVED** [review]

File: src/components/molecules/AdminModal/index.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from 'react'
+import { Modal } from './style'
+
+const AdminModal = ({ show, children, onHide, title }) => {
+  const [showModal, setShowModal] = useState(false)
```
Comment (normandanielmq) [inline src/components/molecules/AdminModal/index.js:5]: Why do we need an internal state for this?

**Review by normandanielmq — APPROVED** [review]

**Pull request merged**